cmd/snap-preseed/preseed_linux.go

```diff
@@ -103,6 +106,30 @@ var systemSnapFromSeed = func(rootDir string) (string, error) {
 	return coreSnapPath, nil
 }
 
+const snapdPreseedSupportVer = `2.44.0`
```

Comment:
We could use 2.43.3+ as the minimal supported version. 2.43.3 is smaller than 2.43.3+ but 2.43.3+gitXYZ is bigger:
```
$ diff --git a/strutil/version_test.go b/strutil/version_test.go
index 479381e5c0..174238b234 100644
--- a/strutil/version_test.go
+++ b/strutil/version_test.go
@@ -99,6 +99,9 @@ func (s *VersionTestSuite) TestVersionCompare(c *C) {
 
                // broken
                {"0--0", "0", 0, fmt.Errorf(`invalid version "0--0"`)},
+               // for Pawel
+               {"2.43.3", "2.43.3+", -1, nil},
+               {"2.43.3+git", "2.43.3+", 1, nil},
        } {
                res, err := strutil.VersionCompare(t.A, t.B)
                if t.err != nil {
$ go test
OK: 40 passed
PASS
ok  	github.com/snapcore/snapd/strutil	0.032s
```
Of course if we would release 2.43.4 then we would have to update this version check to 2.34.4+ here etc.

This would also eliminate the need to check for +git explicitly as the above is just a property of how version compare in debian/ubuntu works (and which our version compare models).

Reply:
I love it, thanks for the suggestion!

Reply:
Done.